apps/ima/translations/src/en/validation.json

```diff
@@ -75,6 +75,25 @@
   "has-address": {
     "required": "You must select an option"
   },
+  "house-number": {
+    "required": "Enter your house number/name, and street name",
+    "notUrl": "Please do not enter a link/url into your answers"
+  },
+  "street": {
+    "notUrl": "Enter a valid street name"
+  },
```

Comment:
The validation message for `"house-number"` mentions adding a street name, but there is no required validation for `"street"`. Is this correct?

Reply:
I see - so the field labels are in the new format now, so this validation is ok.

Reply:
We have address line 1 essential. Address line 2 is optional. So we need both house number and street name in address line 1
Correct error message for notUrl added